stdlib/public/core/Integers.swift.gyb

```diff
@@ -2938,6 +2938,7 @@ ${assignmentOperatorComment(x.operator, True)}
 #endif
 // end of FIXME(integers)
 
+@available(swift, deprecated: 4.2, obsoleted: 5)
```

Comment:
Nit: It works up here, but this should be below the documentation comment and indented to match `@inlinable`.